src/pages/new/index.tsx

```diff
@@ -72,7 +72,7 @@ export const whatsNewQuery = groq`*[_type == 'resource' && slug.current == "what
     	path,
       'byline': meta,
     	image,
-      'background': images[label == 'banner-background'][0].url,
+      'background': images[label == 'banner-image-blank'][0].url,
```

Comment:
I reverted this back to `banner-image-blank` as that's the label we have been using for the rest of the courses that have banners